src/main/java/edu/wpi/grip/core/operations/opencv/PointGeneratorOperation.java

```diff
@@ -0,0 +1,45 @@
+package edu.wpi.grip.core.operations.opencv;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import org.bytedeco.javacpp.opencv_core.Point;
+
+public class PointGeneratorOperation implements CVOperation {
+
+    private final SocketHint<Number> xHint = new SocketHint<Number>("x", Number.class, -1, SocketHint.View.SPINNER, new Integer[]{Integer.MIN_VALUE, Integer.MAX_VALUE}, false);
+    private final SocketHint<Number> yHint = new SocketHint<Number>("y", Number.class, -1, SocketHint.View.SPINNER, new Integer[]{Integer.MIN_VALUE, Integer.MAX_VALUE}, false);
+    private final SocketHint<Point> outputHint = new SocketHint<Point>("point", Point.class, Point::new, SocketHint.View.NONE);
+
+    @Override
+    public String getName() {
+        return "Point";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Creates a Point";
```

Comment:
I would name this "New Point" and word the description as "Create a point".  Same for Size.